### ISSUE TITLE
Zipkin plugin uses incorrect value for khs log

### ISSUE THREAD

**Issue by evertvanaart** [Kong/kong#9153]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

Kong 2.8.1

### Current Behavior

In the `ZipkinLogHandler:header_filter()` function of the Zipkin plugin, the timestamp for the `khs` log is determined as follows:

```
    local header_filter_start_mu =
      ngx_ctx.KONG_HEADER_FILTER_STARTED_AT and ngx_ctx.KONG_HEADER_FILTER_STARTED_AT * 1000
      or ngx_now_mu()
```

However, `KONG_HEADER_FILTER_STARTED_AT` doesn't exist; a text search shows that this value isn't used anywhere else in the code, most importantly not in the main `init.lua` file. As a result, the above code always falls back on `ngx_now_mu()`, which might not be what is expected; it means we're always logging the time at which the `header_filter()` function of the Zipkin plugin is called, not the time at which Kong's `header_filter` phase started.

It looks like the variable name should have been `KONG_HEADER_FILTER_START`, which is set in `init.lua`.

**Comment by hanshuebner** [comment]
Hello @evertvanaart,

thank you for your bug report and your analysis.  I'm going to open an internal ticket for this as even if the fix is a one-liner, we'll want to add some test coverage to ensure that the feature is working correctly with the fix in place.

Kind regards,
Hans

**Comment by kikito** [comment]
Thanks for the report and the analysis, @evertvanaart!

I gave this a very quick look and it looks like a typo, the correct value should be `KONG_HEADER_FILTER_START` - a simple text search should confirm this. I have open a PR to fix it in #9230.

@hanshuebner , please review
